backend/dal/Services/Concrete/ProjectService.cs

```diff
@@ -799,7 +841,28 @@ public async System.Threading.Tasks.Task<Project> SetStatusAsync(Project project
             originalProject.Status = toStatus;
             originalProject.Metadata = project.Metadata;
             project.CopyRowVersionTo(originalProject);
-            this.Context.CommitTransaction();
+            try
+            {
+                this.Context.CommitTransaction();
+            }
+            catch (DbUpdateConcurrencyException ex)
+            {
+                foreach (var entry in ex.Entries)
+                {
+                    var proposedValues = entry.CurrentValues;
+                    var databaseValues = entry.GetDatabaseValues();
+
+                    foreach (var property in proposedValues.Properties)
+                    {
+                        var proposedValue = proposedValues[property];
+                        var databaseValue = databaseValues[property];
+
+                        proposedValues[property] = proposedValue;
+                    }
+
+                    entry.OriginalValues.SetValues(databaseValues);
+                }
+            }
```

Comment:
This code block is a duplicate of the other 2 above, it be nice the create a method and call it